src/editors/_shared/hooks/DataTypes/types/ObjectType.js

```diff
@@ -3,36 +3,12 @@ import DataType from "../AbstractDataType";
 import { checkIfDefaultOrDisabled } from "./utils";
 
 class ObjectType extends DataType {
-  // Object type properties definition
   key = DATA_TYPES.OBJECT;
   label = "Object";
-  default = "{}";
+  default = {};
 
-  editComponent = (props, mode = "row") => {
-    const editor = {
-      row: _props => this.stringEditComponent(_props, this.default),
-      dialog: this.codeEditComponent
-    };
-    return editor[mode](props);
-  };
-
-  /**
-   * Validate object value
-   * @param {*} value
-   * @returns
-   */
-  validate(value) {
-    return new Promise(resolve => {
-      try {
-        if (checkIfDefaultOrDisabled(value)) {
-          return resolve({ success: true, value });
-        }
-        const parsed = JSON.parse(value);
-        resolve({ success: parsed.constructor === Object });
-      } catch (e) {
-        resolve({ success: false });
-      }
-    });
+  _validate(value) {
+    return value === undefined || value !== null && typeof value === "object" && !Array.isArray(value);
```

Comment:
Same thing here with the mixing of operators...

```suggestion
    return value === undefined || (value !== null && typeof value === "object" && !Array.isArray(value));
```

Reply:
There is priority to && over ||, but if you wish for () to make it clearer, I'm not against.